dolly-backend-app/src/main/java/no/nav/dolly/repository/IdentRepository.java

```diff
@@ -39,13 +39,11 @@
             "and bp.ident is not null and length(bp.ident) = 11")
     List<GruppeBestillingIdent> getBestillingerFromGruppe(@Param(value = "gruppe") Testgruppe testgruppe);
 
-    @Query("select ti " +
-            "from Testident ti " +
+    @Query("select ti from Testident ti " +
             "join BestillingProgress bp on bp.ident = ti.ident " +
             "join Bestilling b on b.id = bp.bestillingId " +
-            "where ti.testgruppe.id = :gruppe_id " +
-            "and bp.bestillingId = (select max(bp.bestillingId) from BestillingProgress bp where bp.ident = ti.ident) " +
```

Comment:
Litt usikker på om det fungerer uten denne, holdt på en god del med queryen og for grupper med flere bestillingIder så fungerte det ikke 😅